eeps/eep-0011.md

```diff
@@ -3,7 +3,7 @@
      except for Unicode support according to EEP 10
     Type: Standards Track
     Created: 04-Jun-2008
-    Erlang-Version: R12B-5
+    Erlang-Version: OTP_R12B-5
```

Comment:
Do we need to change then the `Status` as well?
Currently, it says:
```
Status: Accepted/R12B-3u Proposal is implemented in OTP release R12B-3,
```
but we just wrote the the version is written `OTP_R12B-X`